bindings/ruby/lib/rock_gazebo/syskit/robot_definition_extension.rb

```diff
@@ -163,6 +171,11 @@ def sdf_export_joint(
                 ignore_joint_names: false,
                 position_offsets: []
             )
+                joint_sdfs = sdf_export_resolve_joint_sdf(model_dev, joint_names)
+                joint_names = joint_sdfs.map do |el|
+                    el.full_name(root: model_dev.gazebo_root_model.sdf.parent)
+                end
```

Comment:
I chose to resolve the joint names in here instead of passing a SDF joint to be resolved in rock_gazebo. The issue I had with the latter was that test_rock_gazebo.rb tests in common_models exported joints without using `sdf_export_joint` and it was VERY troublesome to inject the SDF there the same way we can do for the links.